### PR TITLE
Assorted Doc Build Fixes

### PR DESCRIPTION
**Pull request by Cadair**
*incoherent screaming*

### CODE REVIEW

**Comment by nabobalis** [comment]
It seems me editing the labels, overwrote yours.

**Review by nabobalis — APPROVED** [review]

File: examples/plotting/magnetogram_active_regions.py [hunk 1]
```diff
@@ -8,53 +8,31 @@
...
+# Some tables do not have these columns, so we exit the script if they are not
+# present.
 
 if 'I' in srs_table['ID'] or 'IA' in srs_table['ID']:
     srs_table = srs_table[np.logical_or(srs_table['ID'] == 'I',
                                         srs_table['ID'] == 'IA')]
 else:
-    print("Warning : No I or IA entries for this date.")
-    srs_table = None
+    raise ValueError("No I or IA entries for this date.")
```
Comment (nabobalis) [inline examples/plotting/magnetogram_active_regions.py:35]: Do we will need this?

Comment (Cadair) [reply examples/plotting/magnetogram_active_regions.py:35]: :man_shrugging: It feels better to spell out that some of the files don't have the columns if we are expecting people to edit this script to their own dates.

Comment (nabobalis) [reply examples/plotting/magnetogram_active_regions.py:35]: Sure.

**Comment by nabobalis** [comment]
Helioviewer is down,restarting the doc build wont fix it. You will need to skip the examples.

**Review by nabobalis — APPROVED** [review]

**Pull request merged**